.github/workflows/nodejs.yml

```diff
@@ -13,6 +13,7 @@ jobs:
     - run: npm ci --no-audit
     - run: npm run lint --if-present
     - run: npm test
+    - run: npm run make-badges --if-present
```

Comment:
It is entirely my fault and but could you please remove this line as we do not need to run this script in Github action based on @rgwozdz earlier's suggestion.